virttest/qemu_storage.py

```diff
@@ -60,6 +62,16 @@ def filename_to_file_opts(filename):
                 if matches.group('conf') is not None:
                     # optional option
                     file_opts['conf'] = matches.group('conf')
+            elif (matches.group('pool') is not None
```

Comment:
It might be better to merge this code block into last 'if - block'

if (matches.group('pool') is not None
                    and matches.group('image') is not None):

pool and image are required options for rbd backend, while namespace is optional, so we can handle namespace in the same way as 'conf', like
if matches.group('namespace') is not None:
     xxxx

Reply:
thanks for the advice, fixed.